src/frontend/src/pages/AdminToolsPage/AdminToolsFinanceConfig.tsx

```diff
@@ -1,11 +1,12 @@
-import { Grid } from '@mui/material';
-import PageBlock from '../../layouts/PageBlock';
+import { Grid, Typography } from '@mui/material';
 import VendorsTable from './FinanceConfig/VendorsTable';
 import AccountCodesTable from './FinanceConfig/AccountCodesTable';
 
 const AdminToolsFinanceConfig: React.FC = () => {
   return (
-    <PageBlock title="Finance Config">
+    <div>
+      <Typography variant="h4">Finance Config</Typography>
```

Comment:
get rid of this text entirely i don't think we need it since its identical to the tab selector 'finance configuration'

Reply:
I think we should keep it? Unless the other tab selectors also require it to be removed, I feel like it's better to keep it consistent across for now.